drizzlepac/haputils/processing_utils.py

```diff
@@ -346,3 +346,54 @@ def make_section_str(str="", width=60, symbol='-'):
     side_dash = symbol * (dash_len // 2)
     section_str = "{}{}{}".format(side_dash, str, side_dash)
     return section_str
+
+def find_flt_keyword (hdu, keyword, extname="SCI", extver=1):
+    """Look for the FITS keyword in the Primary and specified extension header
+
+    This routine deliberately does not use astropy.io.fits getval() as it
+    should not be used in application code due to its self-documented inefficiency.
+
```

Comment:
This docstring should make it clear that this function is only designed to work with keywords that return floating point values, as opposed to 'PHOTMODE' which would return a string.